Content.Server/GameObjects/Components/Atmos/GasTankComponent.cs

```diff
@@ -348,9 +392,16 @@ protected override void Activate(IEntity user, GasTankComponent component)
             }
         }
 
-        public void Dropped(DroppedEventArgs eventArgs)
+        [UsedImplicitly]
+        public class ToggleInternalsAction : IToggleItemAction
         {
-            DisconnectFromInternals(eventArgs.User);
+            public void ExposeData(ObjectSerializer serializer) {}
+
+            public void DoToggleAction(ToggleItemActionEventArgs args)
+            {
+                if (!args.Item.TryGetComponent<GasTankComponent>(out var gasTankComponent)) return;
+                gasTankComponent.ToggleInternals();
+            }
```

Comment:
this was a pre-existing flaw in GasTankComponent and HandheldLightComponent. Added a CanUse check for both of them so that it will block regardless of how they try to toggle (via action or via Z or bound UI)